www/src/app/views/InboxContainer.js

```diff
@@ -1,18 +1,16 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import * as actions from '../actions';
 
 import Inbox from '../components/inbox/Inbox';
 import { API_URL } from '../../config';
-import { makeUpdate, get, post } from '../utils';
+import { makeUpdate, post } from '../utils';
 
-export default class InboxContainer extends Component {
+class InboxContainer extends Component {
   constructor(props) {
     super(props);
 
-    const { author } = this.props.params;
-
     this.state = {
```

Comment:
Again, I mostly left this here to be able to quickly do the partial migration, and start the conversation of what this will turn into. There are definitely cases where a component's state are useful. The way I see it is this:

- If it's info shared among different views, or a representation of what's in the DB, it should be set in the redux store and pulled into the view via `mapStateToProps`,
- If it's data/functionality for a simple component, it should be passed through explicitly via a prop on that component,
- if it's data of interest only to that component that is likely to change (edit text input values, for example) it should be a state variable of the component/its container.